### ISSUE TITLE
System.Configuration.ConfigurationManager.AppSettings  is empty even when appsettings files are loaded

### ISSUE THREAD

**Issue by emahama** [dotnet/runtime#77461]

**Comment by steveharter** [comment]
I don't believe `System.Configuration.ConfigurationManager` works with the JSON config file.

@emahama did you obtain sample code? Here's a sample from the official docs: https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.configuration.configurationbuilder

**Comment by emahama** [comment]
This issue can be closed

**Comment by Saibamen** [comment]
> This issue can be closed

Did you find solution how to have appsettings values in `ConfigurationManager`?
